### ISSUE TITLE
_Xfftn very slow on arm64 arch

### ISSUE THREAD

**Issue by IeiuniumLux** [pyFFTW/pyFFTW#286]
If I run the attached `test.ipynb` notebook file (remove .txt) on a [Google Colab](https://colab.research.google.com/drive/151805XTDg--dgHb3-AXJCpnWaqRhop_2) system or the counterpart `test.py` on a local AMD/Intel system with a GeForce GTX 1660 installed, then all calls to the [_Xfftn function](https://github.com/pyFFTW/pyFFTW/blob/master/pyfftw/builders/_utils.py#L138) complete in less than a second.  However, if I run the same `test.py` on a Jetson TX2, then the first time the [_Xfftn function](https://github.com/pyFFTW/pyFFTW/blob/master/pyfftw/builders/_utils.py#L138) is called, it takes more than 90 seconds for it to complete, but subsequent calls are under a second. Does any know why this only happens on an arm64 architecture?

I have built pyFFTW 0.12.0 from source as well as installed via pip3, but the same result.

![test_output](https://user-images.githubusercontent.com/3729432/75131945-ed4d5d00-5689-11ea-9cd3-b502bd55ba42.png)
![lscpu](https://user-images.githubusercontent.com/3729432/75131948-f2121100-5689-11ea-8481-457c5d4f36e2.png)

[profile_trace.txt](https://github.com/pyFFTW/pyFFTW/files/4243294/profile_trace.txt)

[test.py.txt](https://github.com/pyFFTW/pyFFTW/files/4243298/test.py.txt)

[test.ipynb.txt](https://github.com/pyFFTW/pyFFTW/files/4243303/test.ipynb.txt)

**Comment by hgomersall** [comment]
It sounds like the planning stage is taking ages - if so that's a function of the underlying FFTW library. You can work around this by using the [wisdom feature](https://hgomersall.github.io/pyFFTW/pyfftw/pyfftw.html#wisdom-functions) in which previously optimised transforms can be stored and recalled. That would probably be definitive if it works in your case.

**Comment by IeiuniumLux** [comment]
@hgomersall thanks for the suggestion. I'll give the wisdom feature a try.  However, do you have any guess why the planning stage is only slow the first time is run on an arm64 system and not on AMD/Intel systems?

**Comment by hgomersall** [comment]
No, sorry. I haven't really done much on ARM, but if you do find out I'll be interested to know!
